stdlib/pervasives.mli

```diff
@@ -903,6 +943,11 @@ val really_input_string : in_channel -> int -> string
    characters have been read.
    @since 4.02.0 *)
 
+val input_string_all : in_channel -> string
```

Comment:
I find the name of this function ugly. Here are a few alternative proposals `input_contents`, `string_of_in_channel`. `in_channel_to_string`.